EstateAdministrationUI.IntegrationTests/Common/Setup.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace EstateAdministrationUI.IntegrationTests.Common
+{
+    using System;
+    using Ductus.FluentDocker.Services;
+    using Ductus.FluentDocker.Services.Extensions;
+    using NLog;
+    using Shared.Logger;
+    using Shouldly;
+    using TechTalk.SpecFlow;
+
+    [Binding]
+    public class Setup
+    {
+        public static IContainerService DatabaseServerContainer;
+        private static String DbConnectionStringWithNoDatabase;
+        public static INetworkService DatabaseServerNetwork;
+
+        public static String SqlServerContainerName = "shareddatabasesqlserver";
+
+        public const String SqlUserName = "sa";
+
+        public const String SqlPassword = "thisisalongpassword123!";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=5485852)